draft-ietf-jsonpath-base.md

```diff
@@ -1477,9 +1456,9 @@ Since bracket notation is more general than dot notation, it is used to construc
 Single quotes are used to delimit string member names. This reduces the number of characters that
 need escaping when Normalized Paths appear as strings (which are delimited with double quotes) in JSON texts.
 
-The syntax of Normalized Paths is restricted so that there is one and only one way of representing any
-given Normalized Path. Putting this another way, for any two distinct Normalized Paths, a JSON value exists
-that will yield distinct results when the Normalized Paths are applied to it.
+The syntax of Normalized Paths is restricted so that there is one and only one way of identifying a given node.
+Putting this another way, two distinct Normalized Paths are never equivalent to each other: there will always be a JSON value
```

Comment:
@goessner How about this?
```suggestion
Putting this another way, two distinct Normalized Paths are never equivalent to each other: there will always be at least one JSON value
```

Reply:
hmm ... a little better. The more I think about this I tend more towards formulating 1459 sharper, so that we can skip 1460 all together.

Proposal:

`The syntax of Normalized Paths is restricted so that any Normalized Path identifies one and only one single given node.`